pysb/bng.py

```diff
@@ -49,14 +52,25 @@ class BngBaseInterface(object):
 
     @abc.abstractmethod
     def __init__(self, model=None, verbose=False, cleanup=False,
-                 output_prefix=None, output_dir=None):
+                 output_prefix=None, output_dir=None,
+                 model_additional_species=None,
+                 model_population_maps=None):
+        self._logger = get_logger(__name__,
+                                  model=model,
+                                  log_level=verbose)
+        if model:
```

Comment:
Ah, I see. However, `self.verbose` isn't even used after the latest changes to this branch. Can this code just be deleted altogether? :)

FYI, it looks like `LoggerAdapter` does expose `isEnabledFor` (it just propagates to `self.model` which will then work even if `model` itself is another `LoggerAdapter`). In the future we should probably try to implement this sort of thing via `isEnabledFor`.